global.json

```diff
@@ -16,7 +16,7 @@
     "Microsoft.DotNet.Arcade.Sdk": "5.0.0-beta.20309.1",
     "Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk": "5.0.0-beta.20309.1",
     "Microsoft.DotNet.Helix.Sdk": "5.0.0-beta.20309.1",
-    "FIX-85B6-MERGE-9C38-CONFLICT": "1.0.0",
+    "Microsoft.FIX-85B6-MERGE-9C38-CONFLICT": "1.0.0",
```

Comment:
What is this merge conflict entry for? Do we still need it?

Reply:
It was used to avoid merge conflicts when updates to the IL.SDK entry were made in one PR and updates to the Arcade Sdks above that conflict line where made in another PR. We don't need it anymore as the IL.Sdk unfortunately isn't live updating anymore.

Reply:
Thanks: https://github.com/dotnet/runtime/pull/44709